eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -640,4 +640,4 @@ jobs:
         PathtoPublish: $(Build.ArtifactStagingDirectory)/manifests/$(Agent.JobName).xml
         ArtifactName: VerticalManifests
       displayName: Publish Vertical Manifest
-      condition: succeededOrFailed()
+      condition: succeeded()
```

Comment:
The condition could be removed as succeeded is the default.  It would make the task more consistent with the rest.